gen_cert.py

```diff
@@ -510,7 +534,9 @@ def _generate_certificate(
                         designation=designation,
                     )
                 achievements_string = value['credits']
-                achievements_description_string = unicode(self.cert_data['CREDITS'])
+                achievements_description_string = unicode(
+                    self.cert_data.get('CREDITS', '')
```

Comment:
Reason I was asking if the **have** to be defined, is that if they're not defined using `.get()` will set credits the '' without exploding...